packages/graphql/tests/tck/directives/cypher/filtering/cypher-filtering-one-to-one-relationship.test.ts

```diff
@@ -0,0 +1,1591 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Neo4jGraphQL } from "../../../../../src";
+import { createBearerToken } from "../../../../utils/create-bearer-token";
+import { formatCypher, formatParams, translateQuery } from "../../../utils/tck-test-utils";
+
+describe("cypher directive filtering - One To One Relationship", () => {
+    test("1 to 1 relationship", async () => {
+        const typeDefs = /* GraphQL */ `
+            type Movie @node {
+                title: String
+                released: Int
+                actor: Actor!
+                    @cypher(
+                        statement: """
+                        MATCH (this)-[:ACTED_IN]->(actor:Actor)
+                        RETURN actor
+                        """
+                        columnName: "actor"
+                    )
+            }
+
+            type Actor @node {
+                name: String
+                movie: Movie!
+                    @cypher(
+                        statement: """
+                        MATCH (this)-[:ACTED_IN]->(movie:Movie)
+                        RETURN movie
+                        """
+                        columnName: "movie"
+                    )
+            }
+        `;
+
+        const query = `
+            query {
+                movies(where: { actor: { name: "Keanu Reeves" } }) {
```

Comment:
implicit filters are deprecated now, writing here but the same comment is valid elsewhere
```suggestion
                movies(where: { actor: { name_EQ: "Keanu Reeves" } }) {
```